apps/studio.giselles.ai/package.json

```diff
@@ -10,7 +10,7 @@
 		"dev-debug": "LOGLEVEL=debug pnpm run dev",
 		"build": "next build --turbopack",
 		"start": "next start",
-		"check": "biome check --write .",
```

Comment:
studio has "check" script but all packages have "format" and the root turbo command is also "format"